website/static/css/custom.css

```diff
@@ -99,15 +99,3 @@ article iframe {
   margin-right: auto;
   max-width: 100%;
 }
-.getAPI {
-  color: #0073e6; 
-  font-weight: bold;
-}
-.postAPI {
-  color: #00bf7d; 
-  font-weight: bold;
-}
-.deleteAPI {
-  color: #f49200; 
-  font-weight: bold;
-}
```

Comment:
hmm... idk why this happened, must've been when running mvn commands to spellcheck before I committed

Reply:
do you want to just restore the changes to this file?